### PR TITLE
Whos turn

### PR DESCRIPTION
**Pull request by waders92**
So I added a "turn" field to the game. (schema.rb)
When a game is created, the player that created it will have the first turn.  I created 2 games in my local and you will see that the starting turn is different based on who created the game.  So I'm confident my code does what it should.

I am having a heck of a time to get the test to pass.  It says -
`ActionController::UrlGenerationError:
       No route matches {:action=>"update", :controller=>"games", :game=>{:turn=>19283}}`

If someone could look at my test, that would be boss, then I can move onto switching turns. 

screen shot 2017-03-07 at 4 21 25 pm

screen shot 2017-03-07 at 4 22 30 pm

### CODE REVIEW

File: spec/controllers/games_controller_spec.rb [hunk 1]
```diff
@@ -57,4 +57,16 @@
...
   end
+    it 'sets the first turn to the player that created the game' do
+      player1 = FactoryGirl.create(:player)
+      sign_in player1
+      post :create, params: { game: { turn: '' } }
+      @game = Game.last
+      player2 = FactoryGirl.create(:player)
+      sign_in player2
+
+      patch :update, params: { game: { turn: player1.id  } }
```
Comment (SijiaDavis) [inline spec/controllers/games_controller_spec.rb]: you don't need to update the game manually here. you should remove this line I believe

Comment (SijiaDavis) [inline spec/controllers/games_controller_spec.rb]: also... i think it makes more sense to set turn to be player1's turn when the game is just created?

Comment (waders92) [inline spec/controllers/games_controller_spec.rb]: Ok thanks I'll give it a try.  I was just following the flow of the games#controller.  Thanks for taking a look!

**Comment by waders92** [comment]
This PR is not done, pushed it so Celia could take a look, thanks.

File: db/migrate/20170307170001_add_turn_to_games.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddTurnToGames < ActiveRecord::Migration[5.0]
+  def change
+    add_column :games, :turn, :integer
```
Comment (mscoutermarsh) [inline db/migrate/20170307170001_add_turn_to_games.rb]: If you do something like `active_player_id`

Then you could use an association 'belongs_to :active_player, class_name: 'User'`

Then `game.active_player` would return the user for you.

**Pull request merged**